android/src/toga_android/widgets/internal/pickers.py

```diff
@@ -42,4 +43,6 @@ def create(self):
     def rehint(self):
         self.interface.intrinsic.width = at_least(300)
```

Comment:
No, for the same reason as above. And since date and time pickers look quite different on each platform, I think it makes sense for the backend to have its own value in this case, rather than defining a _MIN_WIDTH that they all share.